core/src/main/java/hudson/model/ManageJenkinsAction.java

```diff
@@ -35,14 +35,15 @@
 @Extension(ordinal=100) @Symbol("manageJenkins")
 public class ManageJenkinsAction implements RootAction {
     public String getIconFileName() {
-        if (Jenkins.get().hasAnyPermission(Jenkins.MANAGE, Jenkins.SYSTEM_READ))
-            return "gear2.png";
-        else
-            return null;
+        return "gear2.png";
```

Comment:
Maybe we can use the Magnifying glass icon for Read Only? might just look better overall